pkg/enqueue/Consumption/FallbackSubscriptionConsumer.php

```diff
@@ -41,13 +36,13 @@ public function consume(int $timeout = 0): void
              * @var callable $processor
              */
             foreach ($this->subscribers as $queueName => list($consumer, $callback)) {
-                $message = $consumer->receiveNoWait();
+                $message = 1 === $subscriberCount ? $consumer->receive((int) ($timeout * 1000)) : $consumer->receiveNoWait();
```

Comment:
[timeout should be milliseconds already](https://github.com/queue-interop/queue-interop/blob/master/src/SubscriptionConsumer.php#L9)

Reply:
It was previously converted to seconds, but I misunderstood the previous operation done on timeout :)

Reply:
which operation are you referring to?

Reply:
https://github.com/php-enqueue/enqueue-dev/pull/626/files/f524a099908afa284278ef337b4042339094bf6b#diff-5f17e0abec53c03fec471b2c5f9f668dL34